dashboard/app/helpers/teacher_application_helper.rb

```diff
@@ -0,0 +1,19 @@
+module TeacherApplicationHelper
+  include Pd::Application::ActiveApplicationModels
+
+  # Returns a user's application from the current year, or nil if no application exists.
+  # Assumes a user has at most one application in a year.
+  def current_application
+    TEACHER_APPLICATION_CLASS.
+      where(application_year: APPLICATION_CURRENT_YEAR).
+      find_by(user: current_user)
+  end
+
+  def has_incomplete_application?
+    current_application && current_application.status == 'incomplete'
+  end
+
+  def has_unreviewed_application?
+    current_application && current_application.status == 'unreviewed'
```

Comment:
this looks unused, will we need it for something soon?

Reply:
Good call––just deleted it (and associated tests)